.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -47,6 +47,9 @@ def get_input(question, default=""):
                                  formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 parser.add_argument('--debug', action='store_true', default=False, help='Configure Submitty to be in debug mode. '
                                                                         'This should not be used in production!')
+parser.add_argument('--setup-for-sample-courses', action='store_true', default=False,
```

Comment:
Given the changes to capability set, I would maybe just reterm this to `--setup-default-capabilities` or something as the list is really just stuff that comes with Submitty by default to even function.

Reply:
its not default (don't want this on production)

Reply:
We may want to consider having this not be in `.setup/CONFIGURE_SUBMITTY.py` then as there's a chance someone will of course use this for their production installation.

But will not block this PR on that discussion.